src/Qir/check-sources-formatted.ps1

```diff
@@ -0,0 +1,31 @@
+#Requires -PSEdition Core
+#Requires -Version 7.1
+
+param (
+    [Parameter()]
+    [String]
+    $DirPath = "$PSScriptRoot"
```

Comment:
`$Path` is a pretty common name for this parameter in most other PowerShell cmdlets (e.g.: `Get-ChildItem`, `Get-Content`, and similar).
```suggestion
    $Path = "$PSScriptRoot"
```